addons/hr_holidays/static/src/avatar_card_popover_patch.xml

```diff
@@ -6,5 +6,10 @@
             <i t-elif="user.im_status === 'leave_away'" class="fa fa-fw fa-plane text-warning" title="Idle" role="img" aria-label="User is idle"/>
             <i t-elif="user.im_status === 'leave_offline'" class="fa fa-fw fa-plane text-700" title="Out of office" role="img" aria-label="User is out of office"/>
         </xpath>
+        <xpath expr="//span[@t-esc='user.name']" position="after">
+            <span class="text-warning me-1 small" t-if="OutOfOfficeDateEndText">
+                <t t-esc="OutOfOfficeDateEndText"/>
+            </span>
+        </xpath>
```

Comment:
`t-if` is the most important pseudo attribute so should be the 1st one even before `class`.